resources/metadata/detectExecuteScan.yaml

```diff
@@ -290,7 +290,7 @@ spec:
         description:
           "This flag determines if the scan is submitted to the server. If set to true, then the scan request is submitted to the server only when changes are detected in the Open Source Bill of Materials
           If the flag is set to false, then the scan request is submitted to server regardless of any changes.
-          For more details please refer to the [documentation](https://github.com/blackducksoftware/detect_rescan/blob/master/README.md)"
+          For more details please refer to the [documentation](https://community.synopsys.com/s/document-item?bundleId=integrations-detect&topicId=properties%2Fconfiguration%2Fblackduck-server.html&_LANG=enus)"
```

Comment:
do not change this. let it remain - https://github.com/blackducksoftware/detect_rescan/blob/master/README.md